src/handlers/button.handler.ts

```diff
@@ -0,0 +1,64 @@
+import {
+    ButtonBuilder,
+    ButtonInteraction,
+    Events,
+    InteractionButtonComponentData,
+} from 'discord.js';
+import { Handler } from '..';
+import { BaseSchema, Output } from 'valibot';
+import { error } from 'console';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const buttonMap = new Map<string, Button<any>>();
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Here aswell

Reply:
I actually think we do

Reply:
but I'm gonna look into it a bit more

Reply:
tried to "fix". Those `any`s are for the greater good

Reply:
btw it is typeface still, the class enforces a certain generic and valibot verifies the id